static-assets/components/cstudio-common/common-api.js

```diff
@@ -5908,7 +5908,7 @@ var nodeOpen = false,
           statusClass = workflowIcons.neverpublished + ' never-published';
         } else if (statusObj.live) {
           //live
-          statusClass = workflowIcons.live + ' live';
+          statusClass = 'live';
```

Comment:
Why change this?

Reply:
@jvega190?

Reply:
workflowIcons.live is undefined (since there is no icon for live), I removed it to avoid 'undefined' as part of the class string